fts5_vs_orama_search_demo.html

```
@@ -0,0 +1,482 @@
+<div class="tabs">
+  <button class="tab-btn active" id="btn-fts5" onclick="showTab('fts5')">⚡ SQLite FTS5</button>
+  <button class="tab-btn" id="btn-orama" onclick="showTab('orama')">◈ Orama search</button>
+</div>
+
+<!-- FTS5 Panel -->
+<div class="panel active" id="panel-fts5">
+  <div class="grid2">
+    <div>
+      <div class="search-row">
+        <input type="text" id="fts-q" value="database" placeholder='e.g. database*, "full text", search OR cache'>
+        <button onclick="runFTS()">Search</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)